geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloFeatureSource.scala

```diff
@@ -32,54 +32,54 @@ import org.locationtech.geomesa.accumulo.process.query.QueryVisitor
 import org.locationtech.geomesa.accumulo.process.stats.StatsVisitor
 import org.locationtech.geomesa.accumulo.process.tube.TubeVisitor
 import org.locationtech.geomesa.accumulo.process.unique.AttributeVisitor
-import org.locationtech.geomesa.utils.geotools.RichSimpleFeatureType.RichSimpleFeatureType
 import org.opengis.feature.FeatureVisitor
 import org.opengis.feature.`type`.Name
 import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
 import org.opengis.filter.Filter
-import org.opengis.filter.expression.{Expression, PropertyName}
+import org.opengis.filter.expression.PropertyName
 import org.opengis.filter.sort.SortBy
 import org.opengis.referencing.crs.CoordinateReferenceSystem
 import org.opengis.util.ProgressListener
 
-import scala.util.Try
 import scala.collection.JavaConversions._
+import scala.util.Try
 
 abstract class AccumuloFeatureSource(val dataStore: AccumuloDataStore, val featureName: Name)
     extends SimpleFeatureSource with LazyLogging {
 
-  import org.locationtech.geomesa.utils.geotools.Conversions._
-
   private[data] val typeName = featureName.getLocalPart
 
   lazy private val hints = Collections.unmodifiableSet(Set.empty[Key])
 
-  // The default behavior for getCount is to use Accumulo to look up the number of entries in
-  //  the record table for a feature.
-  //  This approach gives a rough upper count for the size of the query results.
-  //  For Filter.INCLUDE, this is likely pretty close; all others, it is a lie.
-
-  // Since users may want *actual* counts, there are two ways to force exact counts.
-  //  First, one can set the System property "geomesa.force.count".
-  //  Second, there is an EXACT_COUNT query hint.
+  override lazy val getSchema: SimpleFeatureType = dataStore.getSchema(featureName)
```

Comment:
ah, no it used to be a def but I figured we only need to compute it once so I made it a lazy val.